custom/polio_rdc/ucr/data_sources/users.json

```diff
@@ -0,0 +1,291 @@
+            },
+            {
+                "column_id": "administrateur",
+                "comment": null,
+                "create_index": false,
+                "datatype": "string",
+                "display_name": "administrateur",
+                "expression": {
+                    "datatype": null,
+                    "property_path": [
+                        "user_data",
+                        "administrateur"
+                    ],
+                    "type": "property_path"
+                },
+                "is_nullable": true,
+                "is_primary_key": false,
+                "transform": {},
+                "type": "expression"
+            },
+            {
+                "column_id": "formateur",
+                "comment": null,
+                "create_index": false,
+                "datatype": "string",
+                "display_name": "formateur",
+                "expression": {
+                    "datatype": null,
+                    "property_path": [
+                        "user_data",
+                        "formateur"
+                    ],
+                    "type": "property_path"
+                },
+                "is_nullable": true,
+                "is_primary_key": false,
+                "transform": {},
+                "type": "expression"
+            },
+            {
+                "column_id": "recruteur",
+                "comment": null,
+                "create_index": false,
+                "datatype": "string",
+                "display_name": "recruteur",
+                "expression": {
+                    "datatype": null,
+                    "property_path": [
+                        "user_data",
+                        "recruteur"
+                    ],
+                    "type": "property_path"
+                },
+                "is_nullable": true,
+                "is_primary_key": false,
+                "transform": {},
+                "type": "expression"
+            },
+            {
+                "column_id": "superviseur",
+                "comment": null,
+                "create_index": false,
+                "datatype": "string",
+                "display_name": "superviseur",
+                "expression": {
+                    "datatype": null,
+                    "property_path": [
+                        "user_data",
+                        "superviseur"
+                    ],
+                    "type": "property_path"
+                },
+                "is_nullable": true,
+                "is_primary_key": false,
+                "transform": {},
+                "type": "expression"
+            },
+            {
+                "column_id": "count",
+                "comment": null,
+                "create_index": false,
+                "datatype": "small_integer",
+                "display_name": "count",
+                "expression": {
+                    "constant": 1,
```

Comment:
To be summed in the report config, I assume?

Reply:
yep